From 04590183fc347dbd5f9a049192534c0c889db53e Mon Sep 17 00:00:00 2001
From: Michael Scott Cuthbert <cuthbert@mit.edu>
Date: Sat, 23 May 2015 15:27:25 -0400
Subject: [PATCH] small tab fix

---
 js/midi/plugin.webmidi.js | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/js/midi/plugin.webmidi.js b/js/midi/plugin.webmidi.js
index 03d1c07..33e244b 100644
--- a/js/midi/plugin.webmidi.js
+++ b/js/midi/plugin.webmidi.js
@@ -78,14 +78,14 @@
 			plugin = access;
 			var pluginOutputs = plugin.outputs;
 			if (typeof pluginOutputs == 'function') { // Chrome pre-43
-			  output = pluginOutputs()[0];
+				output = pluginOutputs()[0];
 			} else { // Chrome post-43
-        output = pluginOutputs[0];
+				output = pluginOutputs[0];
 			}
 			if (output === undefined) { // nothing there...
-			  errFunction();
+				errFunction();
 			} else {
-			  opts.onsuccess && opts.onsuccess();			
+				opts.onsuccess && opts.onsuccess();			
 			}
 		}, errFunction);
 	};
